com.ibm.wala.cast.python/source/com/ibm/wala/cast/python/ir/PythonCAstToIRTranslator.java

```diff
@@ -596,7 +631,8 @@ protected boolean doVisit(CAstNode n, WalkContext context, CAstVisitor<WalkConte
 		    context.setValue(n, result);
 			return true;
 		
-		} else if(n.getKind() == CAstNode.GLOBAL_DECL){
+		}
+		else if(n.getKind() == CAstNode.GLOBAL_DECL){
```

Comment:
Could we revert these formatting changes so that conflicts could be more easily resolved?